tests/cache/tests.py

```diff
@@ -302,11 +304,10 @@ def custom_key_func(key, key_prefix, version):
 
 
 def caches_setting_for_tests(base=None, exclude=None, **params):
-    # `base` is used to pull in the memcached config from the original settings,
-    # `exclude` is a set of cache names denoting which `_caches_setting_base` keys
-    # should be omitted.
-    # `params` are test specific overrides and `_caches_settings_base` is the
-    # base config for the tests.
+    # `base` is used to pull in the memcached config from the original
+    # settings, `exclude` is a set of cache names denoting which
+    # `_caches_setting_base` keys should be omitted. `params` are test specific
+    # overrides and `_caches_settings_base` is the base config for the tests.
```

Comment:
I can't decide if this was meant to be a list (and the autofixer mangled it) or just running text